src/components/ScreenSplit.jsx

```diff
@@ -27,7 +46,7 @@ const ScreenSplit = ({ left=(<></>), right=(<></>), centered = false }) => {
       )}
       {right && (
         <Grid item xs={12} lg={6}>
-          <Box component="div" style={styleGridItem}>
+          <Box component="div" className={classes.gridItem} style={styleGridItem}>
```

Comment:
I think we should use either style or className on single component, not both

Reply:
Style is for all components always class is the responsive stuff.